tutorials/R/Geospatial-skills/primer-raster-r/Introduction-to-Raster-Data-In-R/Introduction-to-Raster-Data-In-R.Rmd

```diff
@@ -311,6 +315,8 @@ plot(DEM, col=col, breaks=brk, main="DEM with a Custom (but flipped) Legend", le
 # Third, turn xpd back on to force the legend to fit next to the plot.
 par(xpd = TRUE)
 
+#Run the entire code block to plot* 
```

Comment:
You can remove this comment please - most users will just be copying/pasting code from the website code sections into their console/script and running there, so they have no perception of code chunks like we do in the Rmarkdown docs